desktop/src/main/java/bisq/desktop/main/dao/bonding/BondingViewUtils.java

```diff
@@ -101,18 +105,29 @@ private void lockupBond(byte[] hash, Coin lockupAmount, int lockupTime, LockupRe
                             Consumer<String> resultHandler) {
         if (GUIUtil.isReadyForTxBroadcast(p2PService, walletsSetup)) {
             if (!DevEnv.isDevMode()) {
-                BSFormatter formatter = new BSFormatter();
-                String duration = formatter.formatDurationAsWords(lockupTime * 10 * 60 * 1000L, false, false);
-                new Popup<>().headLine(Res.get("dao.bond.reputation.lockup.headline"))
-                        .confirmation(Res.get("dao.bond.reputation.lockup.details",
-                                bsqFormatter.formatCoinWithCode(lockupAmount),
-                                lockupTime,
-                                duration
-                        ))
-                        .actionButtonText(Res.get("shared.yes"))
-                        .onAction(() -> publishLockupTx(hash, lockupAmount, lockupTime, lockupReason, resultHandler))
-                        .closeButtonText(Res.get("shared.cancel"))
-                        .show();
+                try {
+                    Tuple2<Coin, Integer> miningFeeAndTxSize = daoFacade.getMiningFeeAndTxSize(lockupAmount);
+                    Coin miningFee = miningFeeAndTxSize.first;
```

Comment:
Unfortunately daoFacade.getMiningFeeAndTxSize is not correctly named. It is only valid for the blind vote tx (I will rename the method in another commit). We need to use the correct tx which will be created at the lockup (without triggering any side effects as the user can cancel).